### ISSUE TITLE
WP5.7 - Photo Essay block editor styles are broken making block unusable

### ISSUE THREAD

**Issue by acketon** [bu-ist/bu-blocks#325]
![image](https://user-images.githubusercontent.com/3139096/225116977-6ec16fcf-76bc-4034-b2c6-129e95c72156.png)

**Comment by sksaju** [comment]
Hi, @acketon I think this issue is a duplicate of https://github.com/bu-ist/bu-blocks/issues/308? should I continue to dig into this or do you want to take it? as you already assigned yourself! please let me know. Thanks

**Comment by acketon** [comment]
@sksaju yeah I'll take this one since I think it is just CSS issues. And yeah it looks like it is a duplicate of #308
